scripts/allowlist/README.md

```diff
@@ -0,0 +1,79 @@
+hexadecimal string. If the account has a nominated validator, the account and the validator hexadecimal strings need
+to be separated by a comma:
+
+    0x5B38Da6a701c568545dCfcB03FcB875f56beddC4, 0xAb8483F64d9C6d1EcF9b849Ae677dD3315835cb2
+    0xCA35b7d915458EF540aDe6068dFe2F44E8fa733c
+    0x4b20993Bc481177ec7E8f571ceCaE8A9e22C02db, 0x79731D3Ca6b7E34aC0F824c42a7cC18A495cabaB
+
+The two lines above specify three accounts on the allow list, as well as the validators for the first and third account.
+
+### Run the Script
+
+To run the script in the `scripts/allowlist` directory: `node createContent.js`
+###Output
+
+The script creates a file named `Storage.txt`. The content of this file for the example above will look like this:
+
+    "0x<Address of Contract>": {
+      "balance": "0x00",
+      "code": "0x<Contract Code>"
+      "storage": {
+        "0000000000000000000000000000000000000000000000000000000000000000": "0000000000000000000000000000000000000000000000000000000000000002",
+        "290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563": "000000000000000000000000ab8483f64d9c6d1ecf9b849ae677dd3315835cb2",
+        "290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e564": "00000000000000000000000079731d3ca6b7e34ac0f824c42a7cc18a495cabab",
+        "36306db541fd1551fd93a60031e8a8c89d69ddef41d6249f5fdc265dbc8fffa2": "0000000000000000000000000000000000000000000000000000000000000101",
+        "58d9a93947083dcdedec58d43912ce0326f251a85b7701c5de5bc7d7a150676e": "0000000000000000000000000000000000000000000000000000000000000001",
+        "e20f19dc6931eb9e42fe3f21abe1a9ef59942d8e586871d88564d0d0b63a5e5c": "0000000000000000000000000000000000000000000000000000000000010101",
+        "0000000000000000000000000000000000000000000000000000000000000002": "0000000000000000000000000000000000000000000000000000000000000003"
+      },
+      "version": "0x01"
+    }
+
+The content of the file needs to be placed in the genesis file for the network. In addition the <_Address of Contract_> 
+and <_Contract Code_> need to be filled in.
+
+An example of a genesis file using QBFT can be found in the `genesis.json` file in this directory.
+
+* The <_Address of Contract_> needs to be the same that is stated in the _qbft_ section of the genesis file for the _validatorcontractaddress_.
+* The <_Contract Code_> needs to contain the binary runtime code for the `ValidatorSmartContractAllowList.sol` contract in `contracts/allowlist`.  
+  The binary can be found in the example genesis.json file in this directory. For this binary the 
+`ValidatorSmartContractAllowList.sol` contract was compiled using --_bin-runtime_ and --_optimize_ options of the solidity compiler
+
+General information about the genesis file can be found here: https://besu.hyperledger.org/en/stable/Reference/Config-Items/  
+
+#### Short Description of the Content of the "storage" Section in the above Example Output
```

Comment:
This section is surprisingly...detailed.

Reply:
Do you think it is too much? 
What I was thinking was, that people implementing their own management contracts might need some detail to understand how to do create the storage content. But I guess they could always check out the script ...
I'll make it a lot shorter :-)

Reply:
Yep!